framework/sandstone_tests.cpp

```diff
@@ -3,6 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <charconv>
+#include <fstream>
+#include <iostream>
```

Comment:
Please don't include `<iostream>` because of how it generates pre-`main()` code.
https://github.com/gcc-mirror/gcc/blob/releases/gcc-14.1.0/libstdc%2B%2B-v3/include/std/iostream#L78-L83